packages/module/src/ConsoleInternal/components/catalog/_catalog.scss

```diff
@@ -51,7 +51,7 @@ $catalog-tile-width: $pfext-m-catalog-tile-width;
   }
 
   &__kind-label {
-    font-weight: var(--pf-v5-global--FontWeight--bold);
+    font-weight: var(--pf-v6-global--FontWeight--bold);
```

Comment:
for bold body text, the token is [`--pf-t--global--font--weight--body--bold`](https://github.com/patternfly/patternfly/blob/ceb0aa1a538064c928727d7051792bf07ca6ff78/src/patternfly/base/tokens/tokens-local.scss#L25)